sdk/core/azure-core/CHANGELOG.md

```diff
@@ -20,8 +24,9 @@
   - Changed input parameter type of `WithDeadline()` to `Azure::DateTime`.
 - Removed `Azure::Core::PackageVersion`.
 - Removed from `Azure::Core::Http::Policies` namespace: `HttpPolicyOrder`, `TransportPolicy`, `RetryPolicy`, `RequestIdPolicy`, `TelemetryPolicy`, `BearerTokenAuthenticationPolicy`, `LogPolicy`.
+- Removed `Azure::Core::Context::HasKey()`.
```

Comment:
Add "- Removed `HasKey()`." to "Changes to `Azure::Core::Context`:" section (line 22)